src/components/IntroLinear/StoryLine/StoryStepper.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import {Typography } from "@material-ui/core";
+
+import MobileStepper from "@material-ui/core/MobileStepper";
+
+import Button from "@material-ui/core/Button";
+
+
+const StoryStepper = ({ timesClicked, handleNextClick, handleBackClick }) => {
+
+    const [activeStep, setActiveStep] = React.useState(0);
```

Comment:
Is it possible to create the activeStep state variable in the parent component StoryLine, instead of having both timesClicked and activeStep? It seems like they are always synced up so I think it would be cleaner to share that variable between the 2 components by having it be passed down from StoryLine.